electrode_models/air_electrode.py

```diff
@@ -69,45 +76,45 @@ def __init__(self, input_file, inputs, sep_inputs, counter_inputs,
         if 'host_form' not in inputs or inputs['host_form'] ==  'sphere':
             self.A_init = self.eps_host * 3./self.r_host
         elif inputs['host_form'] == 'cylinder':
-            self.A_init = self.eps_host * 2./self.r_host  
+            self.A_init = self.eps_host * 2./self.r_host
         else:
             raise ValueError("Support host_form values include: 'sphere'",
                 " or 'cylinder'.")
 
         if 'host_surf_frac' in inputs:
             self.A_init *= inputs['host_surf_frac']
-        # For some models, the elyte thickness is different from that of the 
+        # For some models, the elyte thickness is different from that of the
         # electrode, so we specify it separately:
         self.dy_elyte = self.dy
 
         # Inverse double layer capacitance, per unit interfacial area.
         self.C_dl_Inv = 1/inputs['C_dl']
 
-        # Microstructure-based transport scaling factor, based on Bruggeman 
+        # Microstructure-based transport scaling factor, based on Bruggeman
         # coefficient of -0.5:
-        self.elyte_microstructure = self.eps_elyte_init**1.5 # where would we use this?
-        
-        # SV_offset specifies the index of the first SV variable for the 
+        self.elyte_microstructure = self.eps_elyte_init[0]**1.5 #This is used by the
+
+        # SV_offset specifies the index of the first SV variable for the
         # electode (zero for anode, n_vars_anode + n_vars_sep for the cathode)
         self.SV_offset = offset
 
         # Determine the electrode capacity (Ah/m2)
 
-        # Max voume concentration of the product species (assuming all 
+        # Max voume concentration of the product species (assuming all
         # electrolyte has been replaced by product species)
         stored_species = inputs['stored-species']
         v_molar_prod = \
             self.product_obj[stored_species['name']].partial_molar_volumes[0]
 
         self.capacity = (stored_species['charge']*ct.faraday
-                * self.eps_elyte_init * inputs['thickness']
+                * self.eps_elyte_init[0] * inputs['thickness']
```

Comment:
I think this needs to reflect the distribution of porosities.  If we assume that the discretized layers are all of uniform thickness, you can just take the average eps_elyte_init.